dotcom-rendering/.storybook/mocks/log4js.js

```diff
@@ -0,0 +1,13 @@
+// @ts-check
+
+/** This is a mock logger to replace [logging.ts][]
+ *
+ * [logging.ts]: ../../dotcom-rendering/src/server/lib/logging.ts
```

Comment:
I’m assuming this link still works properly?

Reply:
you assume wrong :( updated now, good spot